spec/lib/telephony/otp_sender_spec.rb

```diff
@@ -7,6 +7,13 @@
     Telephony::Test::Call.clear_calls
   end
 
+  shared_examples 'pinpoint valid SSML message' do
+    it 'does not contain reserved SSML characters' do
+      # See: https://docs.aws.amazon.com/polly/latest/dg/escapees.html
+      expect(Nokogiri::XML(message).text).not_to match(/["&'<>]/)
```

Comment:
bonus: make Nokogiri strict so we can catch any non-well formed XML? [source](https://nokogiri.org/tutorials/ensuring_well_formed_markup.html#strict-well-formedness)

```suggestion
      expect(Nokogiri::XML(message) { |config| config.strict }.text).not_to match(/["&'<>]/)
```

Reply:
> bonus: make Nokogiri strict so we can catch any non-well formed XML? [source](https://nokogiri.org/tutorials/ensuring_well_formed_markup.html#strict-well-formedness)

Sure, that's a good idea.